server/types/doc.go

```diff
@@ -0,0 +1,53 @@
+/*
+Package types contains all available entities that compose Docktor application.
+These entities are structures persited to database and exposed within a REST-like API.
+
+Daemon and Site
+
+Docker is used in Docktor to deploy web tools automatically. A daemon defines an entrypoint to a Docker daemon. A site defines where the daemon is located for statistical purposes.
+This daemon can be opened or secured with TLS configuration.
+
+User
+
+A user is someone who is registered and authorized to use Docktor. The person can be either administrator or simple user. An administrator is able to do and see everything. Rights of simple user are defined in groups to whom they belong to.
```

Comment:
`can be either` -> `can either be`
`to whom` -> `to which` car c'est au groupe que les users appartiennent, pas à une personne